### PR TITLE
add telemetry for missing files

### PR DESCRIPTION
**Pull request by elahehrashedi**
add telemetry to report the missing Binary/JSON files from the extension folder.

### CODE REVIEW

File: Extension/src/common.ts [hunk 1]
```diff
@@ -573,6 +576,7 @@ export async function checkInstallBinariesExist(): Promise<boolean> {
                 if (!await checkFileExists(file)) {
                     installBinariesExist = false;
                     console.log(`Extension file ${file} is missing.`);
+                    Telemetry.logLanguageServerEvent("missingBinary", { "source": `${file}` });
```
Comment (bobbrow) [inline Extension/src/common.ts]: You should use `path.basename(file)` instead. We can't log absolute paths in telemetry.

File: Extension/src/common.ts [hunk 2]
```diff
@@ -603,6 +607,7 @@ export async function checkInstallJsonsExist(): Promise<boolean> {
         if (!await checkFileExists(path.join(extensionPath, file))) {
             installJsonsExist = false;
             console.log(`Extension file ${file} is missing.`);
+            Telemetry.logLanguageServerEvent("missingJson", { "source": `${file}` });
```
Comment (bobbrow) [inline Extension/src/common.ts:611]: Same here. Just log the name of the file instead of the full path.

**Review by bobbrow — CHANGES_REQUESTED** [review]

File: Extension/src/common.ts (hunk 1 above)
Comment (elahehrashedi) [reply Extension/src/common.ts]: fixed.

File: Extension/src/common.ts (hunk 2 above)
Comment (elahehrashedi) [reply Extension/src/common.ts:611]: the file name used here already a base name (not a full path)

**Review by bobbrow — APPROVED** [review]

**Review by sean-mcmanus — APPROVED** [review]

**Pull request merged**